.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,39 @@
+name: Node benchmarks
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 0 1 * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12
+    - name: benchmarks
+      run: |
+        npm install
+        npm start y 100 10 40
+        npm start y 100 10 40
+        node_version=$(node --version)
+        benchmark_title=$(cat << EOF
+        # Benchmarks
+        * __Machine:__ $(uname -a) | $(node -r os -p "\`\${os.cpus().length} vCPUs | \${Math.ceil(os.totalmem() / (Math.pow(1024, 3)))}GB\`").
+        * __Method:__ \`autocannon -c 100 -d 40 -p 10 localhost:3000\` (two rounds; one to warm-up, one to measure).
+        * __Node:__ \`$node_version\`
+        * __Run:__ $(date)
+        EOF)
+        benchmark_table=$(node benchmark-compare.js -t -c)
+        strip_readme=$(node -r fs -p 'fs.readFileSync("./README.md", "utf-8").split(/# Benchmarks/)[0]')
+        git checkout master
+        echo -e "${strip_readme:?}\n${benchmark_title:?}\n\n${benchmark_table}" > README.md
+        git add README.md
+        git config user.name 'Github Actions'
+        git config user.email '<>'
+        git commit -m "Add new benchmarks to README.md"
+        git remote set-url --push origin https://${{ secrets.GITHUB_USERNAME }}:${{ secrets.GITHUB_REPO_TOKEN }}@github.com/fastify/benchmarks
+        git push
```

Comment:
I’m a bit worried in having a commit pushed to master every day. Can we figure out a slightly different mechanism (maybe at every change of package-lock?)

Reply:
This actions is run once a month, not on every commit.